### DISCUSSION TITLE
Improve experience upgrading configs to new defaults

### DISCUSSION THREAD

**Discussion by hseg** [d99kris/nchat#244] [Ideas]
Was just bitten by #238 -- I hadn't caught the advisory to update my
keybindings, and couldn't figure out why the ones documented in the manual
wouldn't work. Two features could've helped here:
- [https://neomutt.org/guide/gettingstarted#2-5-%C2%A0help](A la neomutt),
  provide a help screen listing all available commands and the keys currently
  bound to them (and preferably, list its keybinding first in the help bar).
- [https://weechat.org/files/doc/stable/weechat_user.en.html#upgrade](A la
  weechat), provide an invocation to upgrade configurations to new defaults
  (potentially just warning about differences from defaults, or auto-upgrading
  those values that were unchanged from default)

**Comment by d99kris** [comment]
Hi @hseg - thanks for suggesting new features. Ensuring all key bindings are listed in the help bar is something that's been on my own todo-list, so I hope to check that eventually. The other suggested functionalities may also be considered.

FYI - I'll move this issue to a feature request.
